README.md

```diff
@@ -119,6 +119,15 @@ Serve files relative to `path`.
 Byte offset at which the stream starts, defaults to 0. The start is inclusive,
 meaning `start: 2` will include the 3rd byte in the stream.
 
+##### caseSensitive
+
+When set to `true` all the requests against the file system are made case sensitive even on case insensitive file systems.
+This effects not only the file name but the whole path.
```

Comment:
(this is just a comment) I assume by "whole path" that includes what was provided also as the `root` option? If so, it should probably call that out, as there is the input named `path`, which may be mistake for "the whole path" in this wording.